### ISSUE TITLE
test failing due to FELIX parameter changes

### ISSUE THREAD

**Issue by ibaned** [gahansen/Albany#291]
http://cdash.sandia.gov/CDash-2-3-0/testDetails.php?test=3652165&build=70426

```
O_GIS_GisAdjointSensitivityStiffeningBasalFriction (Failed) 
```
```
p=0: *** Caught standard std::exception of type 'Teuchos::Exceptions::InvalidParameterName' :

 Error!  The parameter "Bed Topography Name" does not exist
 in the parameter (sub)list "FELIX Basal Friction Coefficient".
 
 The current parameters set in (sub)list "FELIX Basal Friction Coefficient" are:
 
   {
     "Sliding Velocity QP Variable Name" : string = sliding_velocity
     "BF Variable Name" : string = BF basalside
     "Effective Pressure QP Variable Name" : string = effective_pressure
     "Bed Roughness Variable Name" : string = bed_roughness
     "Side Set Name" : string = basalside
     "Coordinate Vector Variable Name" : string = Coord Vec basalside
     "Parameter List" : ParameterList* = 0x11133a58
     "Physical Parameter List" : ParameterList* = 0x1112e408
     "Stereographic Map" : ParameterList* = 0x13244ad8
     "Bed Topography QP Name" : string = bed_topography
     "Thickness QP Name" : string = ice_thickness
     "Basal Friction Coefficient Variable Name" : string = beta
   }
```

**Comment by bartgol** [comment]
Whoops, my bad. Will push a fix today.

**Comment by bartgol** [comment]
Fixed by commit 9140627
